packages/spinners/package.json

```diff
@@ -12,7 +12,7 @@
   "bugs": {
     "url": "https://github.com/trufflesuite/truffle/issues"
   },
-  "version": "0.1.2",
+  "version": "0.2.0-0",
```

Comment:
Oh, interesting, actually changing this in the PR itself... we haven't typically done this, but it might be a good idea for us to start!